### DISCUSSION TITLE
CREATE EXTENSION TO GENERATE MODEL CLASSES IN PHP

### DISCUSSION THREAD

**Discussion by devfabriciobr** [microsoft/vscode-discussions#116] [Extension Development QnA] [answered]
Hello.. I'm wanting to develop an extension in visual studio code where I can make a connection with the MySQL database, then list all the tables in that database.. after that select the tables and finally the extension generate the classes model of each table in PHP. The thing is, I don't know how to create this extension.. can anyone help me?

Fabricio
Whatsapp: https://wa.me/5583986556461

**Comment by gjsjohnmurray** [comment, marked as answer]
I suggest you start at https://code.visualstudio.com/api/get-started/your-first-extension
